packages/office-ui-fabric-react/src/components/Button/examples/Button.ContextualMenu.Example.tsx

```diff
@@ -1,116 +1,56 @@
+  const { disabled, checked } = props;
 
-export class ButtonContextualMenuExample extends React.Component<IButtonProps, {}> {
-  public render(): JSX.Element {
-    const { disabled, checked } = this.props;
+  return (
+    <DefaultButton
+      text="New item"
+      iconProps={addIcon}
+      menuProps={menuProps}
+      // Optional callback to customize menu rendering
+      menuAs={_getMenu}
+      // Optional callback to do other actions (besides opening the menu) on click
+      onMenuClick={_onMenuClick}
+      // By default, the ContextualMenu is re-created each time it's shown and destroyed when closed.
+      // Uncomment the next line to hide the ContextualMenu but persist it in the DOM instead.
+      // persistMenu={true}
+      allowDisabledFocus
+      disabled={disabled}
+      checked={checked}
+    />
+  );
+};
 
-    return (
-      <div className={css(classNames.twoup)}>
-        <div>
-          <Label>Non persisted menu</Label>
-          <DefaultButton
-            data-automation-id="test"
-            disabled={disabled}
-            allowDisabledFocus={true}
-            checked={checked}
-            iconProps={{ iconName: 'Add' }}
-            menuAs={this._getMenu}
-            text="New"
-            // tslint:disable-next-line:jsx-no-lambda
-            onMenuClick={ev => {
-              console.log(ev);
-            }}
-            menuProps={{
-              items: [
-                {
-                  key: 'emailMessage',
-                  text: 'Email message',
-                  iconProps: { iconName: 'Mail' }
-                },
-                {
-                  key: 'calendarEvent',
-                  text: 'Calendar event',
-                  iconProps: { iconName: 'Calendar' }
-                }
-              ],
-              directionalHintFixed: true
-            }}
-          />
-        </div>
-        <div>
-          <Label>Persisted menu</Label>
-          <DefaultButton
-            data-automation-id="test"
-            disabled={disabled}
-            allowDisabledFocus={true}
-            checked={checked}
-            persistMenu={true}
-            iconProps={{ iconName: 'Add' }}
-            menuAs={this._getMenu}
-            text="New"
-            // tslint:disable-next-line:jsx-no-lambda
-            onMenuClick={ev => {
-              console.log(ev);
-            }}
-            menuProps={{
-              items: [
-                {
-                  key: 'emailMessage',
-                  text: 'Email message',
-                  iconProps: { iconName: 'Mail' }
-                },
-                {
-                  key: 'calendarEvent',
-                  text: 'Calendar event',
-                  iconProps: { iconName: 'Calendar' }
-                }
-              ],
-              directionalHintFixed: true
-            }}
-          />
-        </div>
-      </div>
-    );
-  }
+function _getMenu(menuProps: IContextualMenuProps): JSX.Element {
```

Comment:
Build is complaining because `menuProps` here shadows the `menuProps` variable declared in line 10.